### PR TITLE
add unittests for find_delta_to functions

### PR DESCRIPTION
**Pull request by justy989**
-make multiple commands more readable

### CODE REVIEW

**Comment by justy989** [comment]
I'll need help from @ryabouza building this on osx tomorrow I guess, I tried %i and %d, but clearly something else is going to be required to fix it.

**Comment by ryabouza** [comment]
You'll need to use the PRI\* macros in inttypes.h.

File: ce_config.c [hunk 1]
```diff
@@ -1701,33 +1701,44 @@ bool key_handler(int key, BufferNode* head, void* user_data)
...
                                    assert(cmd[cmd_len-1] == NEWLINE);
                                    cmd[cmd_len-1] = 0;
 
-                                   ce_append_line(config_state->view_current->buffer_node->buffer, cmd);
+                                   ce_append_line(command_buffer_node->buffer, cmd);
                               }
 
-                              pclose(pfile);
+                              // append the return code
+                              snprintf(cmd, BUFSIZ, "exit code: %d", WEXITSTATUS(pclose(pfile)));
```
Comment (robber-m) [inline ce_config.c]: Assign the exit status of `pclose` to a variable, then use `WEXITSTATUS(exit_status)` in the `snprintf` call

**Review by robber-m — CHANGES_REQUESTED** [review]

File: ce_config.c (hunk 1 above)
Comment (justy989) [reply ce_config.c]: That worked !

**Review by robber-m — APPROVED** [review]

**Pull request merged**